history/characters/120000_nraqi.txt

```diff
@@ -52,6 +52,24 @@
 	1.1.1={ birth=yes add_trait = creature_nraqi add_trait = being_void }
 	861.6.23={ death=yes }
 }
+120105={
+	name=Zakajz
+	give_nickname = nick_the_corruptor
+	dynasty=90100
+	dna="iebmbjggfip"
+	culture=nraqi religion=old_gods_worship
```

Comment:
Has N'raqi portrait at the start. Hopefully know way to fix it.
```suggestion
	culture=cthraxxi religion=old_gods_worship
```

Reply:
fixed